setup.py

```diff
@@ -37,4 +37,14 @@ def package_data(pkg, roots):
         ]
     },
     package_data=package_data("crowdsourcehinter", ["static", "public"]),
+    classifiers=[
+        'Framework :: Django :: 1.11',
+        'Programming Language :: Python',
+        'Programming Language :: Python :: 2',
+        'Programming Language :: Python :: 2.7',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3.6',
```

Comment:
It looks like we're not actually testing the 3.6 support yet; could you uncomment that from the Travis configuration while you're at it?

Reply:
That was done in #3 but I added it here as well.

Reply:
Thanks, didn't see that one; I'll see about getting that cleaned up.